scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -1099,13 +1099,13 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <reporter> elements.
   //
   private class ReporterElement {
-    private var rtype     : String = null
-    private var config    : String = null
-    private var filename  : String = null
-    private var directory : String = null
-    private var classname : String = null
+    private var rtype     : String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.